environment/db-seed.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
This script will be used in Codebuild to seed the database in e2e-test environment.